fairseq/model_parallel/models/pipeline_parallel_transformer/model.py

```diff
@@ -561,10 +605,10 @@ def __init__(
     ):
         super().__init__(dictionary)
         self.register_buffer("version", torch.Tensor([3]))
-        try:
-            from fairscale.nn import Pipe
-        except ImportError:
-            raise ImportError("Please install fairscale with: pip install fairscale")
+        if TORCH_PIPE:
+            logger.info('Using torch pipe')
+        else:
+            logger.info('Using fairscale pipe')
```

Comment:
Earlier, the ImportError was thrown on use. With this change, just importing this file is sufficient to cause an import error.

Reply:
@pritamdamania87 - could you add the import statements within the `__init__()` ?